utils/config.py

```diff
@@ -61,6 +61,34 @@ def get_deployer_account() -> Union[LocalAccount, Account]:
     return accounts.load(os.environ["DEPLOYER"]) if (is_live or "DEPLOYER" in os.environ) else accounts[4]
 
 
+def get_web3_storage_token() -> str:
+    is_live = get_is_live()
+    if is_live and "WEB3_STORAGE_TOKEN" not in os.environ:
+        raise EnvironmentError("Please set WEB3_STORAGE_TOKEN env variable to the web3.storage api token")
```

Comment:
Maybe provide a hint on how to bypass the check if it's needed :thinking:

Reply:
Added more info

Reply:
Added more info